src/pythainer/examples/builders/__init__.py

```diff
@@ -241,6 +241,64 @@ def vulkan_builder() -> PartialDockerBuilder:
     return builder
 
 
+def vTune_builder(
+    lib_dir: str = "/home/${USER_NAME}/workspace/libraries",
+) -> PartialDockerBuilder:
+    """
+    Configures a Docker builder for Vtune development.
+    Installs necessary Vtune packages.
+    and prepares the environment variables
+
+    Since Vtune in reality is still profiling the host system you will need
+    to make changes to the host system to get full functionality of Vtune on the docker container.
+
+    Returns:
+        PartialDockerBuilder: A Docker builder ready to use the intel Vtune profiler.
```

Comment:
please capitalize terms properly everywhere. It is "Intel VTune", not "intel Vtune".